codegenerator/cli/npm/envio/package.json.tmpl

```diff
@@ -39,6 +39,8 @@
     "evm.schema.json",
     "fuel.schema.json",
     "rescript.json",
-    "src/**/*.res"
+    "src/**/*.res",
+    "src/**/*.resi",
+    "src/**/*.ts"
```

Comment:
Maybe this would work:
```
src
!src/**/*.bs.js
```
But I'm not sure, so I kept it simple

Reply:
Hmmm, are there some files that we are trying to omit from this package? Maybe it's better to just exclude those. For eg, moving a plain js file in with a binding to it (like db functions for eg) won't work and we need to remember to do this otherwise we only discover it on publish.

Reply:
Yeah, we can add `.npmignore` later. I think we only want to exclude a few files and tests in the future.